docs/CHANGELOG.md

```diff
@@ -1,3 +1,169 @@
+  * **pois**: Whitelist `kind_detail` for `sports` related kinds to just: `10pin`, `9pin`, `american_football`, `archery`, `athletics`, `badminton`, `baseball`, `basketball`, `beachvolleyball`, `billiards`, `bmx`, `boules`, `bowls`, `canoe`, `chess`, `climbing`, `cricket`, `cricket_nets`, `cycling`, `equestrian`, `exercise`, `field_hockey`, `fitness`, `football`, `free_flying`, `futsal`, `gaelic_games`, `golf`, `gymnastics`, `handball`, `hockey`, `horse_racing`, `ice_hockey`, `ice_skating`, `karting`, `model_aerodrome`, `motocross`, `motor`, `multi`, `netball`, `padel`, `pelota`, `rugby`, `rugby_league`, `rugby_union`, `running`, `scuba_diving`, `shooting`, `skateboard`, `skating`, `skiing`, `soccer`, `soccer;basketball`, `softball`, `swimming`, `table_tennis`, `team_handball`, `tennis`, `trampoline`, `volleyball`, `yoga`.
+  * **pois**: Whitelist `kind_detail` for `cousine` related kinds to just: `american`, `asian`, `barbecue`, `breakfast`, `burger`, `cake`, `chicken`, `chinese`, `coffee_shop`, `crepe`, `donut`, `fish`, `fish_and_chips`, `french`, `friture`, `georgian`, `german`, `greek`, `ice_cream`, `indian`, `international`, `italian`, `japanese`, `kebab`, `korean`, `lebanese`, `local`, `mediterranean`, `mexican`, `noodle`, `pizza`, `ramen`, `regional`, `sandwich`, `seafood`, `spanish`, `steak_house`, `sushi`, `tapas`, `thai`, `turkish`, `vegetarian`, `vietnamese`.
+  * **pois**: Whitelist `kind_detail` for `crane` kind to just types: `portal_crane`, `gantry_crane`, `travel_lift`, `floor-mounted_crane`, `shiploader`, `tower_crane`.
+  * **boundaries**: Drop `name`, `name:left`, and `name:right` from `locality` lines at zoom 11 and 12 (Issue [#1738](https://github.com/tilezen/vector-datasource/issues/1738))
+  * **boundaries**: Boundary `name:left` and `name:right` values sometimes flipped, especially for `country` and `region` features (Issue [#1770](https://github.com/tilezen/vector-datasource/issues/1770))
+  * **boundaries**: Made `min_zoom` of `country` lines depend on the data.
+  * **roads**: Fix v1.6 regression where `runway` and `taxiway` lines were sorted below **landuse** layer polygons for the same in error, thanks @bcamper! (Issue [#1814](https://github.com/tilezen/vector-datasource/issues/1814))
+  * **roads**: Expand `is_bridge` to include viaduct and any other not "no" bridges (Issue [#1314](https://github.com/tilezen/vector-datasource/issues/1314))
+  * **earth**: Add `min_zoom` properties. (Issue [#1073](https://github.com/tilezen/vector-datasource/issues/1073))
+  * **water**: Drop all lake name variants zooms 0-4, again. (Issue [#1730](https://github.com/tilezen/vector-datasource/issues/1730))
+  * **water**: Drop additional lake names and variants at mid- and high-zooms (zooms 5-15) based on area (Issue [#1730](https://github.com/tilezen/vector-datasource/issues/1730))
```

Comment:
Merge with line above - wasn't really a separate issue?

Reply:
Fixed in 10796aeb4b6bb82bd19961ca05551838fed9c8d5.